Documentation/network/bgp-control-plane.rst

```diff
@@ -281,7 +281,56 @@ values, graceful restart configuration and others.
 
    Change of an existing neighbor configuration can cause reset of the existing BGP
    peering connection, which results in route flaps and transient packet loss while
-   the session reestablishes and peers exchange their routes.
+   the session reestablishes and peers exchange their routes. To prevent packet loss,
+   it is recommended to configure BGP graceful restart.
```

Comment:
@YutaroHayakawa Is this recommendation okay to give in the document?

Reply:
yeah, makes sense to me :)